src/modules/esl-share/core/esl-share-config.ts

```diff
@@ -1,5 +1,7 @@
-/** {@link ESLShareConfig} provider type definition */
-export type ESLShareConfigProviderType = () => Promise<ESLShareConfig>;
+import defaultConfig from '../config/default.json';
```

Comment:
It looks like it's a strict dependency on the config module 
So that means a user requiring a short config can not reduce the bundle size...
It is not acceptable according to our acceptance criteria:
The user can use the default provided by ESL if he wants, but he should not if he doesn't want to.